exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/OtlpGrpcSpanExporter.java

```diff
@@ -41,13 +49,21 @@
  * <ul>
  *   <li>{@code otel.otlp.span.timeout}: to set the max waiting time allowed to send each span
  *       batch.
+ *   <li>{@code otel.otlp.endpoint}: to set the endpoint to connect to.
+ *   <li>{@code otel.otlp.use.tls}: to set use or not TLS.
+ *   <li>{@code otel.otlp.metadata} to set key-value pairs separated by semicolon to pass as request
+ *       metadata.
  * </ul>
  *
  * <p>For environment variables, {@link OtlpGrpcSpanExporter} will look for the following names:
  *
  * <ul>
  *   <li>{@code OTEL_OTLP_SPAN_TIMEOUT}: to set the max waiting time allowed to send each span
  *       batch.
+ *   <li>{@code OTEL_OTLP_ENDPOINT}: to set the endpoint to connect to.
```

Comment:
There is a proposal in the specs to standardize environment variables for this sort of thing. Can you please add a comment to https://github.com/open-telemetry/opentelemetry-specification/pull/666/files with the needs here?

Reply:
https://github.com/open-telemetry/opentelemetry-specification/pull/666#issuecomment-649615527